sdk/python/kfp/_client.py

```diff
@@ -672,7 +672,7 @@ def _get_workflow_json(self, run_id):
     workflow_json = json.loads(workflow)
     return workflow_json
 
-  def upload_pipeline(self, pipeline_package_path, pipeline_name=None):
+  def upload_pipeline(self, pipeline_package_path, pipeline_name=None, description: str = None):
```

Comment:
nit: can we unify the code style by wrapping '=' with two spaces for `pipeline_name` as well? Same below.

Reply:
Done.